src/Metadata.ts

```
@@ -2,9 +2,9 @@ import { HttpClient, InstanceOptions } from './HttpClient'
 import { BucketMetadata } from './responses'
 
 const routes = {
-  Bucket: (bucket: string) => `/buckets/${bucket}`,
-  Metadata: (bucket: string) => `/buckets/${bucket}/metadata`,
-  MetadataKey: (bucket: string, key: string) => `/buckets/${bucket}/metadata/${key}`,
+  Bucket: (bucket: string) => `/buckets/${process.env.VTEX_APP_NAME}/${bucket}`,
```

This environment variable does not include the app _vendor_. It is a problem, right? What if I have `vtex.banana` and `omnilogic.banana`?

And I think we should throw an error in `Metadata` and `VBase` constructors if that variable is not available, just in case.

---

Yeah, I should be using `VTEX_APP_ID`.